<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fond</class>
 <widget class="QDialog" name="Fond">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1357</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>贷款类别：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_29">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="currentText">
            <string>商业贷款</string>
           </property>
           <item>
            <property name="text">
             <string>商业贷款</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>公积金贷款</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>组合型贷款</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>计算方式：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_28">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="currentText">
            <string>根据贷款总额计算</string>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>根据面积、单价计算</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>根据贷款总额计算</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>贷款总额：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_26">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>请输入贷款总额</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>万</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>按揭年数：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_22">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>25年（300期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24年（288期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23年（276期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22年（264期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21年（252期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20年（240期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19年（228期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18年（216期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17年（204期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16年（192期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年（180期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14年（168期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13年（156期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年（144期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11年（132期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10年（120期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9年（108期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8年（96期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7年（84期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6年（72期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5年（60期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4年（48期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3年（36期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2年（24期）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1年（12期）</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>贷款利率：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_24">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>20年3月20日贷款市场报价利率（LPR）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19年9月20日贷款市场报价利率（LPR）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年10月24日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年10月24日利率上限（1.05倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年10月24日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年10月24日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年10月24日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年8月26日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年8月26日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年8月26日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年8月26日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年6月28日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年6月28日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年6月28日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年6月28日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年5月11日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年5月11日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年5月11日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年5月11日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年3月1日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年3月1日利率上限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年3月1日利率上限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15年3月1日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14年11月22日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14年11月22日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14年11月22日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14年11月22日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年7月6日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年7月6日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年7月6日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年7月6日基准利率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年6月8日利率上限（1.1倍）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年6月8日利率下限（85折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年6月8日利率下限（7折）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12年6月8日基准利率</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_25">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_9">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>4.75</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>还款方式：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_20">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_1">
           <property name="text">
            <string>等额本息</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_21">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>等额本金</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>289</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_1">
           <property name="text">
            <string>开始计算</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_19">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>清空重填</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
